app/Services/Email.php

```diff
@@ -133,8 +134,6 @@ protected function parseLinks($string)
      */
     protected function setupEmail()
     {
-        $leaderboardVars = $this->getLeaderboardVars($this->users);
```

Comment:
Diego cleaned up a lot of that processing stuff, it only processes the leaderboard once and then it gets stored in flash() -- I can play with moving it out if you prefer tho